packages/cli/src/commands/generate/scaffold/templates/components/Name.tsx.template

```diff
@@ -9,6 +9,24 @@ const DELETE_${singularConstantName}_MUTATION = gql`
     }
   }
 `
+const humanize = (value) => {
```

Comment:
Could we reuse the same `humanize` function as we use in `scaffold.js`?

Reply:
I tried but let me see again. I just may not have looked close enough. I'd rather, too.

Reply:
I don't think I can here because that is the "display" function that formats the enum like those that format dates and json etc.

But maybe I can pre-humanize them in the scaffold.js

Reply:
Doesn't seem so straightforward since the display functions take the column name -- not a set of values.

Reply:
I was thinking you could maybe add an import at the top, and then use that. `import humanize from 'humanize-string'`